asistente_ladm_col/utils/decorators.py

```diff
@@ -529,3 +529,20 @@ def decorated_function(*args, **kwargs):
             func_to_decorate(*args, **kwargs)
 
     return decorated_function
+
+
+def _validate_if_plot_is_selected(func_to_decorate):
```

Comment:
We'll remove this one and let the class to check it, along with other dependencies (Java and ReportLib).

Reply:
Thank you for the correction. You are right, we can stop using the decorator, this can be check inside the reports class.